pkg/apis/autoscaling/v1alpha1/pa_validation.go

```diff
@@ -33,14 +33,14 @@ func (pa *PodAutoscaler) Validate(ctx context.Context) *apis.FieldError {
 }
 
 // Validate validates PodAutoscaler Spec.
-func (rs *PodAutoscalerSpec) Validate(ctx context.Context) *apis.FieldError {
-	if equality.Semantic.DeepEqual(rs, &PodAutoscalerSpec{}) {
+func (pa *PodAutoscalerSpec) Validate(ctx context.Context) *apis.FieldError {
+	if equality.Semantic.DeepEqual(pa, &PodAutoscalerSpec{}) {
 		return apis.ErrMissingField(apis.CurrentField)
 	}
-	errs := serving.ValidateNamespacedObjectReference(&rs.ScaleTargetRef).ViaField("scaleTargetRef")
-	errs = errs.Also(rs.ContainerConcurrency.Validate(ctx).
-		ViaField("containerConcurrency"))
-	return errs.Also(validateSKSFields(ctx, rs))
+	errs := serving.ValidateNamespacedObjectReference(&pa.ScaleTargetRef).ViaField("scaleTargetRef")
```

Comment:
you can just chain them.
`return serving.Validate().Also(serving.Validate).Also(...)`

Reply:
we can chain even more, but it's ok.